react/components/SearchBar/components/SearchBar.js

```diff
@@ -41,15 +40,21 @@ const SearchBar = ({
   blurOnSubmit,
   submitOnIconClick,
   minSearchTermLength,
-  autocompleteFullWindow,
+  autocompleteFullWidth,
   intl,
 }) => {
   const container = useRef()
   const { navigate } = useRuntime()
   const handles = useCssHandles(CSS_HANDLES)
   const [searchTerm, setSearchTerm] = useState(inputValue)
   const [inputErrorMessage, setInputErrorMessage] = useState()
-  const inputErrorMessageTimeRef = useRef()
+
+  const messages = defineMessages({
```

Comment:
Please move this out of the component render.

Example:

```js
const messages = defineMessages({
 // ..
})

const SearchBar = ({
```

Doing this we will only create this variable once.

Reply:
Well noticed. Done!